build/common.jl

```diff
@@ -25,17 +35,41 @@ function remote_url(repo_root::AbstractString, name::AbstractString="origin")
     end
 end
 
-const REPO_PATH = abspath(joinpath(@__DIR__, "..", ".."))
-const PKG_URL = remote_url(REPO_PATH)
+function set_url_scheme(url, scheme)
+    m = match(LibGit2.URL_REGEX, url)
+    if m === nothing
+        throw(ArgumentError("URL is not a valid SCP or HTTP(S) URL: $(url)"))
+    end
+    return LibGit2.git_url(; scheme="https", username=something(m[:user], ""),
+                           host=something(m[:host], ""), port=something(m[:port], ""),
+                           path=something(m[:path], ""))
+end
```

Comment:
@glennmoy I can understand your reason for copying the regex from LibGit2 as it's not exported by the library. However, now that I'm using `git_url` as well it seemed better to use the function and regex from the library that tests them.

Reply:
So my understandig is this converts, e.g. `git@github.com:beacon-biosignals/ray.jl` to `https://github.com:beacon-biosignals/ray.jl`?

Reply:
Yes, it converts SCP-like URLs to HTTPS URLs. Specifically `git@github.com:beacon-biosignals/Ray.jl` to `https://github.com/beacon-biosignals/Ray.jl` (there was a syntax issue with your HTTPS URL).